tests/unit/test_coloring.py

```diff
@@ -0,0 +1,200 @@
+"""
+Test file for Coloring class
+"""
+import unittest
+from coloring import *
+from graph_io import load_graph
+
+
+class ColoringCase(unittest.TestCase):
+    coloring = None
+    graph = None
+    v1 = None
+    v2 = None
+
+    @classmethod
+    def create_coloring_helper(self, vertices: List[int], map: dict):
+        coloring = Coloring()
+        for color in map:
+            for value in map[color]:
+                vertex = [v for v in vertices if v.label == value][0]
+                coloring.set(vertex, color)
+        return coloring
+
+    @classmethod
+    def create_graph_helper(self, edges: List[List[int]]):
```

Comment:
Note to self: Volgens mij staat deze methode nu bij veel testen.. Als dit (en je andere PR) gemergd is, zal ik het op een plek zetten zodat het vanaf meerdere plekken gebruikt kan worden.

Reply:
Klopt